src/payment/PaymentPage.jsx

```diff
@@ -90,8 +95,12 @@ class PaymentPage extends React.Component {
           <OrderDetails />
         </div>
         <div className="col-md-7 pl-md-5">
-          <PaymentMethodSelect />
-          <PaymentForm />
+          {isFreeBasket ? <PlaceOrderButton /> : (
```

Comment:
A note for the future: This looks like it changed the way the app works, so that means the semantic commit message should have been fix or feature.  refactor means you didn't change anything visible, you just did the same thing in a different (hopefully better :-) ) way.